config.lua

```diff
@@ -173,6 +172,13 @@ Config.BoatPounds = {
 		Marker = vector3(-738.67, -1400.43, 4.0),
 		Spawner = vector3(-738.33, -1381.51, 0.12),
 		Heading = 137.85
+	},
+
+	Terrorist_Dock = {
+        Marker = vector3(3130.80, 2180.00, 4.40),
+        Spawner = vector3(3112.60, 2173.00, -0.50),
+        Heading = 121.02
+		Hidden = true
```

Comment:
Hidden Config value is not implemented on 
Boat Garages in client.lua 

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/client/main.lua#L1684

Check Line 1718 https://github.com/lockdownstudio/esx_advancedgarage/blob/master/client/main.lua#L1718 on how to implement a Hidden Boat garages.